src/components/CTypeDetails/CTypeDetails.astro

```diff
@@ -46,7 +46,11 @@ const schemaV1 = CType.fromProperties('', {}).$schema;
 const version = schema === schemaV1 ? 'V1' : 'draft-01';
 const kiltCType = CType.fromProperties(title, properties, version);
 
-const { subscan, w3nOrigin } = configuration;
+const { w3nOrigin, indexer } = configuration;
+
+const polkadotExplorerRPC =
+  indexer.polkadotRPCEndpoint +
+  (indexer.polkadotRPCEndpoint.split('.').at(-1) === 'network' ? '' : '%2F');
```

Comment:
Is there a more elegant way to achieve the same end result? I believe at very least we could use `.endsWith('.network')`, but why do we even need to append the magic string?

Reply:
Sadly, polkadot.js handles some rpc-endpoints different than others.

- https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.kilt.io%2F#/explorer/query/67
- https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkilt.dotters.network#/explorer/query/67

Reply:
Would it work to simply have these in the configuration?

    https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.kilt.io%2F#/explorer/query/
    https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkilt.dotters.network#/explorer/query/

Reply:
I just notice that 2 other endpoints are treated differently. 

- https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.api.onfinality.io%2Fpublic-ws#/explorer/query/67
- https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkilt-rpc.dwellir.com#/explorer/query/67

So `.com` as `.network`.

Reply:
The thing is that `%2F` just represents a `/`.  You can actually replace it in the URL and it will work.

Reply:
I believe my proposal can easily accommodate these as well

Reply:
It seems to be just a manner of how polkadot.js was configured. 

This 2 work the same and have the KILT colors: (with `%2F` or `/`)
- https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.kilt.io%2F#/explorer/query/67
- https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.kilt.io/#/explorer/query/67

This one works but have the default polkadot colors: (without `%2F` or `/`)
- https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.kilt.io#/explorer/query/67

Reply:
Maybe is better to get rid of the whole logic about  `%2F` and add a comment to use a trailing `/` for the `kilt.io` endpoints.

Reply:
What do you think of my [suggestion to move the whole URL to the configuration?](https://github.com/galaniprojects/ctypehub/pull/743#discussion_r1713667952)

Reply:
I will be against having the urls hardcoded on the configuration. 

They change and should be manage by our devOps engineer without touching the code. 

This setup also let's you use a local endpoint without first trying it out on polkadot.js.

Reply:
[I guess my idea is best explained in code](https://github.com/galaniprojects/ctypehub/commit/020937945422b5725bc81d9007c3fed81f7fd701)